migration/20201117-remove-all-search-coverage.sql

```diff
@@ -0,0 +1,3 @@
+-- Remove all WorkCoverageRecords pertaining to the search index. This
+-- will force a complete reindex on the next run of bin/search_index_refresh.
+delete from workcoveragerecords where operation='update-search-index';
```

Comment:
Why does this need a reindex?

Reply:
There's a bunch of books in the index that have been indexed with available=null instead of available=false. (Not _all_ books, but a lot of them.) Reindexing them with the new code will make the same books indexed with available=false.